x/concentrated-liquidity/pool_test.go

```diff
@@ -209,7 +212,10 @@ func (s *KeeperTestSuite) TestPoolIToConcentratedPool() {
 
 	// Create default CL pool
 	concentratedPool := s.PrepareConcentratedPool()
-	poolI := concentratedPool.(poolmanagertypes.PoolI)
+	poolI, ok := concentratedPool.(poolmanagertypes.PoolI)
+	if !ok {
+		s.Fail("failed to cast pool to PoolI")
```

Comment:
I think it makes more sense to use `FailNow` when a type conversion error happens
```suggestion
		s.FailNow("failed to cast pool to PoolI")
```